lib/Activity.py

```diff
@@ -764,7 +768,7 @@ def resume(self):
             self.config.logger.error(f"Unable to get activity {self.name} from backend.")
             return
 
-        if last_activity["state"] == 'in_progress' and last_activity["status"] == "Running":
+        if last_activity["state"] == 'in_progress':
```

Comment:
While testing, I noticed not all 'in_progress' states necessarily have a status of 'Running' -- this might be just a fluke on gamora.